### ISSUE TITLE
Screenlayout

### ISSUE THREAD

**Issue by aromero86** [linuxdeepin/developer-center#596]
Hi guys, I have two monitors in vertical  layout (laptop and another monitor). My monitor is bigger than laptop (Monitor: 2560px / Laptop: 1920px). So, I prefer center the monitors but it is imposible.

The monitor configuration tool align two monitors and don't let my center them.

![screenlayout](https://user-images.githubusercontent.com/25776692/44513186-f5a39100-a6bc-11e8-8534-12d53fa898f6.png)
## To Reproduce
Obviously, you need two monitors with diferent resolutions. Yo can easily reproduce it trying to center both monitors.

## Expected behavior
It would be great have both monitors centered.  
![deepinscreenshot_seleccionar area_20180823102119](https://user-images.githubusercontent.com/25776692/44513702-5b444d00-a6be-11e8-8d4e-1b4d4122d57b.png)

With this script created with *arandr*, the problem is fixed:

```
xrandr --output HDMI-1 --off --output DP-1 --mode 2560x1080 --pos 0x0 --rotate normal --output eDP-1 --primary --mode 1920x1080 --pos 296x1080 --rotate normal --output HDMI-2 --off
```
I tried add the script in `/etc/profile.d` and I created a autostart program (at `~/.config/autostart`) but it doesn't work.

## Enviroments:
Deepin 15.7
Xiaomi Notebook Air 13' 1920px
LG monitor Ultrawide 2560px

**PD: Sorry about my poor english**

* Updated for attaching images

**Comment by es02** [comment]
I used to have similar problems when my laptop was my work machine. Keeping it sandwiched between two monitors was painful as aligning by hand often left tiny overlaps or gaps that I couldn't see in the settings dialog.
tl;dr: Better alignment options would be great.

**Comment by meiyixiang** [comment]
Please use our new version, sorry it took so long to deal with your problem
